Simplenote/TagListViewController.swift

```diff
@@ -519,6 +526,17 @@ extension TagListViewController: TagListViewCellDelegate {
 
         present(alertController, animated: true, completion: nil)
     }
+
+    private func shouldRemove(_ tagToRemove: Tag, at indexPath: IndexPath) -> Bool {
```

Comment:
Perhaps, for clarity, this could be named...

- `func verifyTagIsAtIndexPath(tag: Tag, indexPath: IndexPath) -> Bool`

?

Thinking about adding a bit of extra clarity!